lmdeploy/pytorch/backends/cuda/op_backend.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Tuple
+
+import torch
+
+from lmdeploy.pytorch.config import BackendConfig, CacheConfig, ModelConfig
+from lmdeploy.utils import get_logger
+
+from ..base import OpType
+from ..default import DefaultOpsBackend
+
+logger = get_logger('lmdeploy')
+
+
+class CudaOpsBackend(DefaultOpsBackend):
+    """cuda layer backend."""
+
+    @staticmethod
+    def get_name() -> str:
+        """backend name."""
+        raise 'cuda'
```

Comment:
Is it `return` insdead of `raise` ? it seems `get_name()` is not used?

Reply:
Not used. Added just in case.